packages/story-editor/src/components/videoTrim/useVideoTrimMode.js

```diff
@@ -60,16 +61,11 @@ function useVideoTrimMode() {
     if (selectedElement?.type !== 'video' || !selectedElement?.resource) {
       return false;
     }
-    const { isTranscoding, isMuting, isTrimming, local, isExternal } =
-      selectedElement.resource || {};
+    const { resource } = selectedElement;
     return (
       isVideoTrimEnabled &&
       isTranscodingEnabled &&
-      !isTranscoding &&
-      !isMuting &&
-      !isTrimming &&
-      !isExternal &&
-      !local
+      canTranscodeResource(resource)
```

Comment:
This will need to be reworked for trimming since `hasTrimMode` should not return `false` when `isTrimming` or `isMuting` or `isTranscoding`.  But that can be done in #9142.